runtime/json-processor-utils.h

```diff
@@ -0,0 +1,25 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2020 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include <type_traits>
+
+namespace impl_ {
+template<typename T, typename = std::void_t<>>
+struct IsJsonFlattenClass : std::false_type{};
+
+template<typename T>
+struct IsJsonFlattenClass<T, std::enable_if_t<T::json_flatten_class>> : std::true_type{};
+} // namespace impl_
+
+struct JsonRawString {
+  explicit JsonRawString(string &s) noexcept
+    : str(s) {}
+  string &str;
+};
+
+struct JsonEncoderError {
+  static string msg;
```

Comment:
This `string` is static, but never reset between script executions. It must be reset with `hard_reset_var` at a cleanup function. Now It will lead to critical error in runtime:

![image](https://user-images.githubusercontent.com/28536346/175429219-21128d6c-b4fb-4935-ba1d-524ecfe9f4ec.png)